starter-code/routes/drones.js

```diff
@@ -1,20 +1,27 @@
-const express = require('express');
+const express = require("express");
 
 // require the Drone model here
 
 const router = express.Router();
 
-
-router.get('/drones', (req, res, next) => {
+router.get("/drones", (req, res, next) => {
   // Iteration #2
+  Drones.find({}, (err, users) => {
+    if (err) {
+      next(err);
+    } else {
+      const data = {
+        drones: drones
```

Comment:
This will not works,

You defined users as "the result from the database" and you use drones..

change users to drones

Reply:
// always create a data object with key and value:

```
router.get('/', (req, res, next) => {
    Drone.find({}, (err, drones) => {
        if (err) {
            next(err);
        }
        const data = {
            drones: drones,
        }
        res.render('./drones', data);
    });
});
```

Reply:
./drones IF you have a folder called "drones" in "views" folder and a index.ejs file in "drones"